src/cart/crt_internal_types.h

```diff
@@ -119,16 +119,13 @@ struct crt_gdata {
 	volatile unsigned int	cg_refcount;
 
 	/** flags to keep track of states */
-	unsigned int		cg_inited		: 1,
-				cg_grp_inited		: 1,
-				cg_swim_inited		: 1,
-				cg_auto_swim_disable	: 1,
-				/** whether it is a client or server */
-				cg_server		: 1,
-				/** whether scalable endpoint is enabled */
-				cg_use_sensors		: 1,
-				/** whether we are on a primary provider */
-				cg_provider_is_primary	: 1;
+	unsigned int cg_inited : 1, cg_grp_inited : 1, cg_swim_inited : 1, cg_auto_swim_disable : 1,
+	    /** whether it is a client or server */
+	    cg_server              : 1,
+	    /** whether metrics are used */
+	    cg_use_sensors         : 1,
+	    /** whether we are on a primary provider */
+	    cg_provider_is_primary : 1;
```

Comment:
it would be great if you were using `unsigned int` on each field to fix the format or just start using bool instead :)

Reply:
> it would be great if you were using `unsigned int` on each field to fix the format or just start using bool instead :)

we use bitfields here as we have to be careful about the size of this rpc_priv struct. its annoying for formatting though

Reply:
maybe I was not clear, what I'm saying is that you should not separate each field with `,` but instead repeat `unsigned int` for each field so that they remain aligned.
```C
	    /** whether metrics are used */
	    unsigned int cg_use_sensors         : 1;
	    /** whether we are on a primary provider */
	    unsigned int cg_provider_is_primary : 1;
```
does that make sense ?

Reply:
also fwiw this is the `struct crt_gdata` not the rpc_priv struct...

Reply:
> maybe I was not clear, what I'm saying is that you should not separate each field with `,` but instead repeat `unsigned int` for each field so that they remain aligned.
> 
> ```c
> 	    /** whether metrics are used */
> 	    unsigned int cg_use_sensors         : 1;
> 	    /** whether we are on a primary provider */
> 	    unsigned int cg_provider_is_primary : 1;
> ```
> 
> does that make sense ?

Neat, i didnt know it allowed such style